website/light/__init__.py

```diff
@@ -172,11 +172,13 @@ def edit_room( room_id ):
         room.color3 = form.color3.data
         room.last_modify_user = current_user.id
         room.last_modify_time = datetime.now().strftime("%m/%d/%Y %H:%M:%S")
+        room.last_modify_datetime = datetime.now()
         db.session.commit()
         cmd = { 'style':style,'color1':form.color1.data,'color2':form.color2.data,'color3':form.color3.data }
         data_change(cmd, room.session_id)
         return redirect('/home')
```

Comment:
If I made an arbitrary POST request, it won't stop me. You should make it 403 if a request is made within the 30 second buffer

Reply:
Thoughts on leaving that ability open to admin in case you really need to change it for some reason, but locking it for everyone else?